addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -474,6 +476,44 @@ export class Rtc extends Record {
         }, PING_INTERVAL);
     }
 
+    get displaySurface() {
+        return this.state.sourceScreenStream?.getVideoTracks()[0]?.getSettings().displaySurface;
+    }
+
+    showMirroringWarning() {
+        this.state.screenTrack.enabled = false;
+        this.removeMirroringWarning = this.overlay.add(
+            InfiniteMirroringWarning,
+            {
+                onClose: (enableScreenTrack) => {
+                    this.removeMirroringWarning(enableScreenTrack);
+                    this.removeMirroringWarning = null;
+                },
+            },
+            {
+                onRemove: (enableScreenTrack) => {
+                    if (enableScreenTrack) {
+                        this.state.screenTrack.enabled = true;
+                    } else {
+                        this.toggleVideo("screen", false);
+                    }
+                },
+            }
+        );
+        this.state.screenTrack.addEventListener(
+            "ended",
+            () => {
+                if (this.removeMirroringWarning) {
+                    this.removeMirroringWarning();
+                    this.removeMirroringWarning = null;
+                }
+            },
+            {
+                once: true,
+            }
+        );
+    }
```

Comment:
While the stream is being paused, should put a low-opacity centered text on stream to let user know this is being paused. Something like this:


<img width="1901" alt="Screenshot 2025-05-12 at 11 30 16" src="https://github.com/user-attachments/assets/46dcd419-7551-4ad0-aa0e-767d990a204b" />